### PR TITLE
Modal Form for Vuestic UI Landing page (#4265)

### LINKED ISSUE #4265: Modal Form for Vuestic UI Landing page
Develop a modal form designed to collect emails from users interested in a free Vue.js code audit offered by the Vuestic Core Team. The form should engage users effectively, ensure user consent for data processing, and comply with best practices for user interaction.

Requirements:
- - only for https://ui.vuestic.dev/ (landing) not for docs
- Include a close button (X icon) at the top right of the modal.

- Content:
  -- Header: "Get a Free Vue.js Code Audit from Vuestic Core Team".
 -- Text: "Got a Vue.js project and want to make it better? Provide us with some details about your project, and we'll get back to you within 24 hours."
 -- Email input field
 -- CAPTCHA verification
 -- Submit button
 -- Text under the Submit button:
"Click 'Submit!' to consent to processing your data by Epicmax Georgia LLC for marketing purposes, including sending emails. For details, see our Privacy Policy."

Link for privacy: https://www.iubenda.com/privacy-policy/50799764

Functional Requirements:
The modal should not appear immediately upon page load. Implement a delay mechanism like:

<script src="https://cdnjs.cloudflare.com/ajax/libs/js-cookie/2.2.1/js.cookie.min.js"></script>
<script>
const delay = 5; // Pop-up delay in seconds
const popupId = "pop-up-wrapper"; // Pop-up ID name

$(document).ready(function() {
   if (!Cookies.get('alert')) {
     setTimeout(function() {
       document.getElementById(popupId). style.display = "flex";
       Cookies.set('alert', true, { expires: 1 });
     }, delay * 1000);
   }
});
</script>

Ensure the modal respects user privacy and cookie settings to prevent repeated displays to the same user within a given timeframe.

Example: 
image

Image for the right section here:
Got a Vue js project and want to make it better Provide us with some details about your project, and we'll get back to you within 24 hours

### PR DESCRIPTION
**Pull request by Fsss126**
closes #4265

## Description
Added code adit form popup

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/nuxt.config.ts [hunk 1]
```diff
@@ -159,6 +159,7 @@ export default defineNuxtConfig({
   runtimeConfig: {
     public: {
       VITE_STORYBOOK_HOSTNAME: process.env.STORYBOOK_HOSTNAME,
+      recaptchaKey: process.env.NUXT_PUBLIC_RECAPTCHA_KEY,
```
Comment (asvae) [inline packages/docs/nuxt.config.ts:162]: Maybe we can use the same syntax in all cases?

i.e. `NUXT_PUBLIC_RECAPTCHA_KEY`

Right now - there are 3 ways:
* NUXT_PUBLIC_RECAPTCHA_KEY
* RECAPTCHA_SITE_KEY
* recaptchaKey

File: packages/docs/composables/useRecaptcha.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      tabindex?: number;
+      callback?: (token: string) => void;
+      'expired-callback'?: () => void;
+      'error-callback'?: () => void;
+    }
+  ): string;
+  reset(widgetId?: string): void;
+}
+
+const recaptcha = ref<Recaptcha>();
```
Comment (asvae) [inline packages/docs/composables/useRecaptcha.ts:22]: Could you run linter with --fix? We generally don't use `;` in vuestic.

**Review by asvae — APPROVED** [review]
Fouhnd couple of very small issues and we can safely merge.

File: packages/docs/components/Recaptcha.vue [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<template>
```
Comment (asvae) [inline packages/docs/components/Recaptcha.vue:1]: Would recommend a 2 word name for this component.

File: packages/docs/composables/useRecaptcha.ts (hunk 2 above)
Comment (Fsss126) [reply packages/docs/composables/useRecaptcha.ts:22]: Linter doesn't enforce consistent colon code style unfortunately

File: packages/docs/nuxt.config.ts (hunk 1 above)
Comment (Fsss126) [reply packages/docs/nuxt.config.ts]: Done

File: packages/docs/components/Recaptcha.vue (hunk 3 above)
Comment (Fsss126) [reply packages/docs/components/Recaptcha.vue:1]: Would suggest leaving it like this, I respect the guidelines with clear component naming but Recaptcha is clear enough 🙂

**Pull request merged**